clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -706,4 +706,9 @@ public static ApiVersionsResponse createApiVersionsResponse(
                 ApiVersionsResponse.UNKNOWN_FINALIZED_FEATURES_EPOCH,
                 zkMigrationEnabled);
     }
+
+    public static String getCurrentFunctionName() {
```

Comment:
IIRC, we had a discussion about the topic name. What is the updates?

Reply:
Sorry, I missed the newset commend.
Will do ASAP.

Reply:
We can't use `testInfo`, using `testCreateWithDefaults` as example, the name from `testInfo.getDisplayName()` will get `testCreateWithDefaults [1] Type=Raft-Isolated, MetadataVersion=4.0-IV0,Security=PLAINTEXT` and it is illegal name.

Reply:
Have you toke a look at `testInfo.getTestMethod()`?

Reply:
> Have you toke a look at `testInfo.getTestMethod()`?

Yes, they are some problem.

Reply:
> Yes, they are same problem.

Have you test it on local? `testInfo.getTestMethod().get().getName()` show the "pure" method name on my local.

Reply:
Thanks for your info. Update already.